### PR TITLE
Changed UniqueEmail request to GET

### PR DESCRIPTION
**Pull request by robertpin**

### CODE REVIEW

File: ClientApp/viking-vault/src/RegisterForm.tsx [hunk 1]
```diff
@@ -54,15 +54,12 @@ class RegisterForm extends React.Component<any, IFormState> {
...
             headers: {
               'Accept': 'application/json',
               'Content-Type': 'application/json',
-            },
-            body: JSON.stringify({
-              email: this.state.user.email,
-            })
+            }
         }).then(response => response.json()).then(result => {
             if(result === true) { // email is unique
```
Comment (vasi-axinte) [inline ClientApp/viking-vault/src/RegisterForm.tsx]: This comment is not needed.

File: Server/VikingVault.API/Controllers/UniqueEmailController.cs [hunk 2]
```diff
@@ -19,10 +19,10 @@ public UniqueEmailController(IUniqueEmailService uniqueEmailService)
         {
             _uniqueEmailService = uniqueEmailService;
         }
-        [HttpPost]
-        public ActionResult Post([FromBody] UserEmail userEmail)
```
Comment (vasi-axinte) [inline Server/VikingVault.API/Controllers/UniqueEmailController.cs:23]: Did you also delete this model?

**Review by vasi-axinte — CHANGES_REQUESTED** [review]

File: Server/VikingVault.API/Controllers/UniqueEmailController.cs (hunk 2 above)
Comment (robertpin) [reply Server/VikingVault.API/Controllers/UniqueEmailController.cs:23]: Done

File: ClientApp/viking-vault/src/RegisterForm.tsx (hunk 1 above)
Comment (robertpin) [reply ClientApp/viking-vault/src/RegisterForm.tsx]: Done

**Review by vasi-axinte — APPROVED** [review]

**Pull request merged**